packages/grafana-schema/src/veneer/dashboard.types.ts

```diff
@@ -45,33 +44,8 @@ export interface AnnotationContainer extends Omit<raw.AnnotationContainer, 'list
   list?: AnnotationQuery[]; // use the version from this file
 }
 
-export interface Threshold extends Omit<raw.Threshold, 'value'> {
-  // Value represents a lower bound of a threshold. This triggers a visual change in the dashboard when a graphed value is within the bounds of a threshold.
-  // Nulls currently appear here when serializing -Infinity to JSON.
-  value: number | null;
-}
-
-export interface ThresholdsConfig extends Omit<raw.ThresholdsConfig, 'steps'> {
-  steps: Threshold[];
-}
-
-export interface FieldConfig<TOptions = Record<string, unknown>> extends Omit<raw.FieldConfig, 'mappings'> {
+export interface FieldConfig<TOptions = Record<string, unknown>> extends raw.FieldConfig {
   custom?: TOptions & Record<string, unknown>;
-  mappings?: ValueMapping[];
```

Comment:
Likewise this introduces a breaking change. Can you deprecate this field first?

Reply:
It's not really a breaking change. `mappings` is present on the `raw.FieldConfig` and instead of overwriting it here, we use the original type, which is the same as the ones removed here.